packages/opentelemetry-instrumentation-anthropic/opentelemetry/instrumentation/anthropic/__init__.py

```diff
@@ -304,6 +315,12 @@ def _set_token_usage(
         span, SpanAttributes.LLM_USAGE_COMPLETION_TOKENS, completion_tokens
     )
     set_span_attribute(span, SpanAttributes.LLM_USAGE_TOTAL_TOKENS, total_tokens)
+    set_span_attribute(
+        span, "llm.cache_creation_input_tokens", cache_creation_tokens
```

Comment:
Following [convention](https://github.com/traceloop/openllmetry/blob/3cf7a233ed486d4a4f5f1cd8e6c46b1588671792/packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/__init__.py#L47) this should be called `gen_ai.anthropic.usage.cache_creation_input_tokens` (and same for the other one)

Reply:
okay sure! I am fixing it.